src/sql/src/catalog.rs

```diff
@@ -1255,7 +1283,9 @@ impl fmt::Display for CatalogError {
             Self::SchemaAlreadyExists(name) => write!(f, "schema '{name}' already exists"),
             Self::UnknownRole(name) => write!(f, "unknown role '{}'", name),
             Self::RoleAlreadyExists(name) => write!(f, "role '{name}' already exists"),
+            Self::NetworkPolicyAlreadyExists(name) => write!(f, "network policy '{name}' already exists"),
```

Comment:
`trait mz_ore::str::StrExt` has a `quoted()` method that we generally use for this